website/src/releases/2023-11/user-access-for-local-development.mdx

```diff
@@ -0,0 +1,16 @@
+---
+date: 2023-11-09
+title: Introduced granular permissions for Custom Application local development
+description: Custom Application can now be developed by members of a Team with limited permissions.
+type: enhancement
+topics:
+  - Merchant Center
+  - Development
+  - Configuration
+---
+
+You can now have a dedicated team with restricted permissions for local development of your Custom Applications, which was previously available only for members of the `Administrators` Team.
+
+To leverage this feature, ensure your `app-kit` packages are updated to version `22.5.0` or higher, and refer to the guide provided in the [help needed](/help-needed#granular-permissions-for-custom-application-local-development) section within the Custom Application documentation.
```

Comment:
Do you think we should mention here that this is more of a workaround as it has some tradeoffs?

Reply:
Thank you, that's a great point. I've included this information in an infobox